lib/sw-toolbox.js

```diff
@@ -42,5 +42,6 @@ module.exports = {
   options: options,
   cache: helpers.cache,
   uncache: helpers.uncache,
-  precache: helpers.precache
+  precache: helpers.precache,
+  fetchAndCache: helpers.fetchAndCache
 };
```

Comment:
Exposing these in the module exports looks reasonable to me. Wdyt, @jeffposnick? Is there a better path for supporting this?